specifyweb/frontend/locale/README.md

```diff
@@ -0,0 +1,59 @@
+# Instructions on localization
+
+`locale` directory contains a subdirectory for each language in the format
+`<language_code>/LC_MESSAGES/`. Inside `LC_MESSAGES`, there is a `django.po`
+file that contains the `msgid` and `msgstr` for the strings used across the
+back-end files.
+
+According to the convention, `msgid` and `msgstr` have the same
+value for the English language. All additional languages will use the same
+`msgid`, but change `msgstr` to a needed value.
+
+Back-end files can mark strings as translatable by enclosing the string within
+the `_` function.
+
+Example:
+
+```python
+from django.utils.translation import gettext as _
+
+print(_("Some value to translate"))
+```
+
+After adding new translatable strings, run the following command from inside the
```

Comment:
If translatable strings are removed does this step remove them from the `po` file?

Reply:
Yes, they get commended out.
Also, if msgid changes, gettext shows the previous value in a comment


![Screen Shot 2021-07-12 at 10 54 49 PM](https://user-images.githubusercontent.com/40512816/125348148-886cd280-e364-11eb-95ce-b9c52544276d.png)

The only thing I haven't yet figured is hot to receive compilation errors for usages of non-existing keys, but that option must be somewhere out there as gettext supports a lot of customization